Modules/Bridge/NumPy/wrapping/PyBuffer.i.init

```diff
@@ -15,6 +15,16 @@ try:
           if obj is None: return
           self.itk_base = getattr(obj, 'itk_base', None)
 
+      def __reduce_ex__(self, protocol):
+          np_copy = np.array(self, copy=True)
+          return type(self)._reconstruct, (np_copy,), None
+
+      @classmethod
+      def _reconstruct(cls, obj):
+          with memoryview(obj) as m:
+             obj = m.obj
+             if type(obj) is np.ndarray:
+                 return cls(obj, itk_base=None)
```

Comment:
Based on the discussion here:

- https://stackoverflow.com/questions/54486021/how-to-pickle-numpy-ndarray-derived-class
- https://stackoverflow.com/questions/26598109/preserve-custom-attributes-when-pickling-subclass-of-numpy-array

The above two methods should be replaced with something like:

```
def __reduce__(self):
    np_copy = np.array(self, copy=True)
    pickled_state = super(NDArrayITKBase, np_copy).__reduce__()
    return pickled_state
```